src/client/commands/log.rs

```diff
@@ -42,7 +42,7 @@ pub struct CatOpts {
 
     /// Print only specific task output
```

Comment:
`Print only the specified task(s) output. You can use the array syntax to specify multiple tasks.`